baseapp/abci_test.go

```diff
@@ -38,18 +43,18 @@ func TestGetBlockRentionHeight(t *testing.T) {
 		"pruning iavl snapshot only": {
 			bapp: NewBaseApp(
 				name, logger, db, nil,
-				SetPruning(sdk.PruningOptions{KeepEvery: 10000}),
+				SetPruning(sdk.NewPruningOptions(sdk.Nothing)),
 				SetMinRetainBlocks(1),
+				SetSnapshot(snapshotStore, sdk.NewSnapshotOptions(10000, 1)),
 			),
 			maxAgeBlocks: 0,
 			commitHeight: 499000,
-			expected:     490000,
+			expected:     489000,
```

Comment:
This has changed because now we use `snapshot-interval` in place of `pruning-keep-recent`. Although I added `sdk.NewSnapshotOptions(10000, 1))`, snapshot interval is used as another strategy for calculating the block retention height and it just takes precedence over the old strategy used by this unit test.